crates/nargo_cli/tests/test_data/config.toml

```diff
@@ -2,4 +2,10 @@ exclude = []
 
 
 # List of tests (as their directory name) expecting to fail: if the test pass, we report an error.
-fail = ["brillig_assert_fail", "dep_impl_primitive", "workspace_fail", "workspace_missing_toml"]
+fail = [
+    "brillig_assert_fail",
+    "dep_impl_primitive",
+    "depend_on_bin",
+    "workspace_fail",
+    "workspace_missing_toml"
+]
```

Comment:
This breaks the regexes in rebuild.sh

Reply:
Why do we regexp instead of parsing toml? 😛